src/js/bootstrap.js

```diff
@@ -0,0 +1,4153 @@
+      }
+      try {
+        return document.querySelector(e) ? e : null;
+      } catch (t) {
+        return null;
+      }
+    },
+    getTransitionDurationFromElement: function (t) {
+      if (!t) return 0;
+      var n = e(t).css("transition-duration"),
+        i = e(t).css("transition-delay"),
+        o = parseFloat(n),
+        r = parseFloat(i);
+      return o || r
+        ? ((n = n.split(",")[0]),
+          (i = i.split(",")[0]),
+          1e3 * (parseFloat(n) + parseFloat(i)))
+        : 0;
+    },
+    reflow: function (t) {
+      return t.offsetHeight;
+    },
+    triggerTransitionEnd: function (t) {
+      e(t).trigger("transitionend");
+    },
+    supportsTransitionEnd: function () {
+      return Boolean("transitionend");
+    },
+    isElement: function (t) {
+      return (t[0] || t).nodeType;
+    },
+    typeCheckConfig: function (t, e, n) {
+      for (var i in n)
+        if (Object.prototype.hasOwnProperty.call(n, i)) {
+          var o = n[i],
+            r = e[i],
+            s =
+              r && l.isElement(r)
+                ? "element"
+                : null === (a = r) || "undefined" == typeof a
+                ? "" + a
+                : {}.toString
+                    .call(a)
+                    .match(/\s([a-z]+)/i)[1]
+                    .toLowerCase();
+          if (!new RegExp(o).test(s))
+            throw new Error(
+              t.toUpperCase() +
+                ': Option "' +
+                i +
+                '" provided type "' +
+                s +
+                '" but expected type "' +
+                o +
+                '".'
+            );
+        }
+      var a;
+    },
+    findShadowRoot: function (t) {
+      if (!document.documentElement.attachShadow) return null;
+      if ("function" == typeof t.getRootNode) {
+        var e = t.getRootNode();
+        return e instanceof ShadowRoot ? e : null;
+      }
+      return t instanceof ShadowRoot
+        ? t
+        : t.parentNode
+        ? l.findShadowRoot(t.parentNode)
+        : null;
+    },
+    jQueryDetection: function () {
+      if ("undefined" == typeof e)
+        throw new TypeError(
+          "Bootstrap's JavaScript requires jQuery. jQuery must be included before Bootstrap's JavaScript."
+        );
+      var t = e.fn.jquery.split(" ")[0].split(".");
+      if (
+        (t[0] < 2 && t[1] < 9) ||
+        (1 === t[0] && 9 === t[1] && t[2] < 1) ||
+        t[0] >= 4
+      )
+        throw new Error(
+          "Bootstrap's JavaScript requires at least jQuery v1.9.1 but less than v4.0.0"
+        );
+    },
+  };
+  l.jQueryDetection(),
+    (e.fn.emulateTransitionEnd = a),
+    (e.event.special[l.TRANSITION_END] = {
+      bindType: "transitionend",
+      delegateType: "transitionend",
+      handle: function (t) {
+        if (e(t.target).is(this))
+          return t.handleObj.handler.apply(this, arguments);
+      },
+    });
+  var c = "alert",
+    u = e.fn[c],
```

Comment:
Codacy found an issue: [Variable Assigned to Object Injection Sink](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5731813)